src/graphene-simd4f.c

```diff
@@ -1178,10 +1180,10 @@ graphene_simd4f_t
 (graphene_simd4f_reciprocal) (graphene_simd4f_t v)
 {
   graphene_simd4f_t s = {
-    fabsf (v.x) > FLT_EPSILON ? 1.0f / v.x : 0.f,
-    fabsf (v.y) > FLT_EPSILON ? 1.0f / v.y : 0.f,
-    fabsf (v.z) > FLT_EPSILON ? 1.0f / v.z : 0.f,
-    fabsf (v.w) > FLT_EPSILON ? 1.0f / v.w : 0.f
+    1.0f / v.x,
+    1.0f / v.y,
+    1.0f / v.z,
+    1.0f / v.w
```

Comment:
Following a comment from @jadahl, I looked into why this works a bit further and it seems that while IEEE 754 compliance is largely well-adopted by compilers in regards to floating-point division of negative and positive 0, it's still very much compiler-specific behavior as far as the C standard defines.

Specifically, Annex F of the C99 standard (ISO/IEC 9899:1999) defines floating-point division as conforming to the IEEE 754 standard (ISO/IEC 60559:1989). However, since Annex F is largely optional, compilers and C implementations are free to implement only some of it, or none of it.

So to be on the safe side, and to avoid breaking existing users with older or exotic compilers, I think I'll detect scenarios that aren't in conformance and add a fallback implementation which is more likely to be implemented as expected.

Reply:
Done.